src/components/DocumentCard/DocumentCardTitle.scss

```diff
@@ -1,15 +1,14 @@
 @import '../../common/common';
 
-$documentCard-titleSize: 16px;
+$DocumentCard-title-lineHeight: 21px;
 
 .ms-DocumentCardTitle {
   padding: 8px 16px;
   display: block;
   @include ms-font-l;
-  font-size: $documentCard-titleSize;
   color: $ms-color-neutralPrimary;
-  height: 36px; // Two lines of text, making sure the third line is hidden
-  line-height: 20px;
+  height: 38px; // Two lines of text, making sure the third line is hidden
```

Comment:
Really don't like having a fixed height here AND a line height. wouldn't we only want line-height and padding? Your numbers don't add up now. height 38, but padding+lineheight: 37px... does the 1 pixel go on the bottom or top? How about remove height?